tests/later-test.ts

```diff
@@ -74,6 +74,34 @@ QUnit.test('later should rely on stubbed `Date.now`', function(assert) {
   }, 1);
 });
 
+QUnit.test('later shedules timers correctly after time travel', function(assert) {
+  assert.expect(2);
+
+  let bb = new Backburner(['one']);
+  let done = assert.async();
+  let start = originalDateNow();
+  let now = start;
+
+  Date.now = () => now;
```

Comment:
FWIW, we do have `lolex` here, and could use that for a more "proper" time travel test...

Reply:
I'm not sure how to do that, given that I would only want to use it for `Date.now()`, and not `setTimeout()`. Feel free to update it if you have a better way, just make sure it still fails on the old codebase.